keel-web/config/keel.yml

```diff
@@ -70,6 +70,13 @@ okHttpClient:
   readTimeoutMs: 59000
   interceptor.skipHeaderCheck: true
 
+resilience4j.retry:
```

Comment:
nit: I think `retry` should go on the next line.

Reply:
I saw examples for either option - but will change :)